data/languages/fr/text/dialogs.dat

```diff
@@ -1993,6 +1996,7 @@ réputés !?
 dialog{
   id = "chill_valley.bath.hungry_child",
   skip = "all",
+  translators_note = "Reference to episode of the french serie Kaamelott "Ablutions" with Karadoc missing food during his bath",
```

Comment:
The quest editor should not have allowed double quotes here. I guess you can no longer open the file, you need to fix it by hand. I am opening a bug.